epochStart/metachain/systemSCs.go

```diff
@@ -340,9 +342,14 @@ func (s *systemSCProcessor) unStakeOneNode(blsKey []byte, epoch uint32) error {
 		return err
 	}
 
-	peerAccount, errGet := s.getPeerAccount(blsKey)
-	if errGet != nil {
-		return errGet
+	account, errLoad := s.peerAccountsDB.GetExistingAccount(blsKey)
+	if errLoad != nil {
+		return nil
```

Comment:
log.Debug the error?

Reply:
this is not an error. but if account does not exist there is no need to unStake

Reply:
renamed to errExist it is used in multiple places